article/app/views/structuredData/BlogPosting.scala

```diff
@@ -1,4 +1,4 @@
-package views.support.structuredData
+package views.structuredData
```

Comment:
This originally was mirroring common (which has views/support) but if that's not something I should be following then yeah I can definitely pull this code out to an upper level.